docs/workshop/ml_pipeline.md

```diff
@@ -1,3 +1,53 @@
-# Machine Learning Pipeline
+# Create an ML Pipeline using Elyra and Kubeflow Pipelines
 
-This section will consist of Kubeflow UI and demonstrate how we automate the ML workflow from collecting the data, feat eng, exporting tables to trino)
+In order to run the above notebooks in automation, we can configure a Kubeflow Pipeline using the Elyra notebook pipelines editor from Jupyterlab
+
+## Create kubeflow runtime configuration
+
+As a first step, create a Kubeflow runtime configuration. In order to do that
+
+Click the Runtimes icon on the left hand side toolbar. Click the ‘+’ on the top right to create a new runtime configuration and select “New Kubeflow Pipelines Runtime configuration”. Fill in the following:
+
+**Name**: Give the runtime configuration your Github username eg: oindrillac
+**Kubeflow Pipelines API Endpoint:** [http://ml-pipeline-ui.kubeflow.svc.cluster.local/](http://ml-pipeline-ui.kubeflow.svc.cluster.local/)
+**Kubeflow Pipelines Engine:** Tekton
+**Authentication Type:** No Authentication
+**Cloud Object Storage Endpoint:** S3 Endpoint
+**Cloud Object Storage Bucket Name:** S3 Bucket Name
+**Cloud Object Storage Authentication Type:** USER_CREDENTIALS
+**Cloud Object Storage Username: ** S3 Access Key ID
+**Cloud Object Storage Password:** S3 Secret Key
+
+![kubeflow pipeline runtime config](../assets/images/ml-pipeline-kfp-config.png "image_tooltip")
+
+## Configure Pipeline
+
+Open pipeline(end-to-end.pipeline) and click on each node and click on “Open Properties”. Fill in the environment variables which are not pre-filled for each notebook and hit “save” to save the pipeline
+
+![pipeline nodes](../assets/images/ml-pipeline-pipeline-nodes.png "image_tooltip")
```

Comment:
Not sure, if its on my side. But the image looks unclear. wdyt?

Reply:
yeah some of the images can be modified, and also annotated with red boxes. I realize that we also need more screenshots at some places.  Everything may not be perfect as yet, and the doc is still missing some links, but wanted to get this initial draft out so that we can create a barebones jupyterbook. Lets take a note of these enhancements for our next draft.